### DISCUSSION TITLE
LabbyMod Mod Support

### DISCUSSION THREAD

**Discussion by MrButtersDEV** [minecraft-dev/MinecraftDev#1000] [Ideas]
The title sums it up but would it maybe be possible to add support for this mod/client?

**Comment by DenWav** [comment]
I've never heard of this before. How large and active is the developer community?

@Earthcomputer @RedNesto either of you know anything about this?

**Reply by RedNesto** [reply]
I heard of it years ago, I don't know anything about it.

**Reply by DJtheRedstoner** [reply]
Not quite sure what they are referring too here. [LabyMod](https://labymod.net/) is basically an "all in one" forge mod and "client" for 1.8.9, 1.12.2 and 1.16.5. I know there is a [server api](https://github.com/LabyMod/labymod-server-api), but that is just done with plugin messages (don't see what mcdev could do here). There is also an ["addon" api](https://docs.labymod.net/pages/create-addons/introduction/) for the client/mod, but this is just a forge mdk with a few small changes (once again I don't see what mcdev could do here). On a side note laby is notorious for breaking other mods (by overwriting class instance variables instead of just injecting into methods) and stealing code/ideas.

**Reply by Earthcomputer** [reply]
I've never heard of it either, but based on what @DJtheRedstoner said I don't think it is a good idea.
